core/unix/module_elf.c

```diff
@@ -331,12 +331,14 @@ module_fill_os_data(ELF_PROGRAM_HEADER_TYPE *prog_hdr, /* PT_DYNAMIC entry */
     return res;
 }
 
-/* Returned addresses out_base and out_end are relative to the actual
+/* Identifies the bounds of each segment in the ELF at base.
+ * Returned addresses out_base and out_end are relative to the actual
  * loaded module base, so the "base" param should be added to produce
  * absolute addresses.
  * If out_data != NULL, fills in the dynamic section fields and adds
  * entries to the module list vector: so the caller must be
  * os_module_area_init() if out_data != NULL!
+ * Optionally returns the first segment bounds, the max segemnt end, and the soname.
```

Comment:
(Spelling)

Reply:
The problem with coding during the PC meeting...